src/components/wishList/WishListView.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import AlbumGrid from '../AlbumGrid';
+import axiosInstance from '../../apis/axiosClient';
+
+const WishListView = () => {
+  const [albums, setAlbums] = useState();
+  const [wishListId, setWishListId] = useState();
+
+  //fetch wishlist album from API
+  useEffect(() => {
+    const fetchWishlist = async () => {
+      try {
+        const response = await axiosInstance.post(`/wishlist/`); //use axiosInstance to send cookie token with request
+        const wishlistData = response.data.wishlist;
+        setWishListId(wishlistData._id); //set the state for wishlist id
+        setAlbums(wishlistData.albums); //set the state for wishlist albums
+        // Store each album id from wishlist in local storage
+        wishlistData.albums.forEach((album) => {
+          localStorage.setItem(
+            `wishListAlbums-${album._id}`,
+            JSON.stringify(true)
+          );
+        });
```

Comment:
Same suggestion as in `AlbumsList` about potentially updating this to store one single entry in localstorage with an object that has all the wishlist albums, rather than a separate entry for each wishlist album.

Reply:
Done